site/netlify.toml

```diff
@@ -12,3 +12,8 @@
   for = "/favicon.ico"
   [headers.values]
   cache-control = "max-age=604800" # 1 week
+
+[[redirects]]
+from = "/guide/inline-queries.html"
+to = "/plugins/inline-query.html"
+status = 301
```

Comment:
How to add redirects for translations?

Reply:
My words: https://github.com/grammyjs/website/pull/808#discussion_r1239591437

Reply:
```suggestion
[[redirects]]
from = "/*/guide/inline-queries.html"
to = "/:splat/plugins/inline-query.html"
status = 301
```

Reply:
The current entry shouldn’t be removed:

```suggestion
[[redirects]]
from = "/guide/inline-queries.html"
to = "/plugins/inline-query.html"
status = 301

[[redirects]]
from = "/*/guide/inline-queries.html"
to = "/:splat/plugins/inline-query.html"
status = 301
```